### ISSUE TITLE
Can we see EMDB structures in the LiteMol viewer

### ISSUE THREAD

**Issue by sandraorchard** [Complex-Portal/complex-portal-view#60]
I think Sameer said we should be able to

**Comment by tschaka1904** [comment]
I'll double check that.

**Comment by tschaka1904** [comment]
I've tested it, but it doesn't work out of the box. We might need to ask Sameer.

**Comment by bmeldal** [comment]
Go and ask Sameer ;-) Or the developers, they might have a more immediate answer :)

**Comment by tschaka1904** [comment]
I've send an email to Sameer.

**Comment by mandarsd** [comment]
Hi All,

Yes, LiteMol can render EMDB structures :) 

Here is an example EM entry visualization on PDBe entry page http://www.ebi.ac.uk/pdbe/entry/view3D/5mb8/?view=entry_index&viewer=jmol&controls=codename_hero

If you are using the litemol component then the definition will be
`<pdb-lite-mol pdb-id="'5mb8'" hide-controls="true" load-ed-maps="true"></pdb-lite-mol>`

Hope this helps!

Let me know if you have any specific implementation requirement.

Cheers,
Mandar

**Comment by tschaka1904** [comment]
Brilliant! How would that work a well for [EMD-2635](http://www.ebi.ac.uk/pdbe/entry/emdb/EMD-2635)? Or is that not possible yet?

**Comment by mandarsd** [comment]
The current component implementation maps the EMBD entry to its corresponding PDB fitted model structure. That means PDB ID is mandatory.  For example the EMDB entry <a href="/pdbe/entry/emdb/EMD-3459/index">EMD-3459</a> maps to PDB entry <a href="//pdbe.org/5mb8">5mb8</a>. You can obtain this mapping information using EMDB API. 

In case of <a href="/pdbe/entry/emdb/EMD-2635/index">EMD-2635</a> there is no PDB fitted model structure available.

Now if you want to display only the density maps available on EMDB then you'll have to customize the component to render only the EMDB maps. Do you want implement this ?

**Comment by tschaka1904** [comment]
I guess not currently, but maybe in the future. Do you think that this is rather straight forward?

@bmeldal what do you think about: 

> Now if you want to display only the density maps available on EMDB then you'll have to customize the component to render only the EMDB maps. Do you want implement this ?

**Comment by bmeldal** [comment]
Yes, I think we'll leave the density maps for now and focus on what can be implemented using LiteMol in its current version. For recent structures we have both xrefs, the regular PDB one and the EMD-xxxx, so that'll work.

@mandarsd would you be the one to talk to about any regular reciprocal updates? Either way, I need to arrange a meeting with however deals with that for our next step in the Complex Portal - PDB relationship :)

**Comment by mandarsd** [comment]
@tschaka1904 Yes it is pretty straight forward. I'll create an example demonstration with this use case. It will be useful for other developers who want to customize the component.

@bmeldal it depends on the type of PDBe service, for example anything related to web components at PDBe ultimately routes to me and things related to API are forwarded to relevant API developers. I guess Sameer will decide who will be attending the upcoming meetings but for web component updates you can write directly to me :)

**Comment by bmeldal** [comment]
Thanks, Mandar, it'll be API related first. I'll ping Sameer.

**Comment by tschaka1904** [comment]
@bmeldal in this case we will have EMDB IDs which can be displayed with the LiteMol viewer and some can not. How do you want to display that?

**Comment by bmeldal** [comment]
Good question.
I think we'll have to display the PDB IDs with the LiteMol viewer and then have a separate table below for the EMD-xxxx ACs that are just hyperlinked to the entry in the EMDB (link needs fixing as we know). They have different external DB instances in our DB (wwpdb vs emdb).

E.g.:
http://www.ebi.ac.uk/intact/complex/details/EBI-9024024 - just one of each
http://www.ebi.ac.uk/intact/complex/details/EBI-12654601 - lots of PDB subsets and full EM structures.
(neither available in GH instance though).

**Comment by bmeldal** [comment]
Opened Jira:
https://intact.atlassian.net/browse/CP-242

**Comment by bmeldal** [comment]
Keeling open as we need this discussion.

**Comment by tschaka1904** [comment]
Preferably make a decision if you want to continue this effort or not, as it has been hanging around for a bit.
